src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -280,7 +258,6 @@ suspend fun blockingApplicationLogicBehandlingsutfall(
             )
             handleMessageBehandlingsutfall(validationResult, sykmeldingsid, database, loggingMeta)
         }
-        delay(100)
```

Comment:
Vet vi hvorfor denne var her..?

Reply:
Regner med det er for å ikke få 100% CPU forbruk når man ikke bruker .poll med Duration

Reply:
Ser du har lagt på denne igjen, var den der av en annen grunn også? 😅